src/components/Header/index.js

```diff
@@ -1,3 +1,3 @@
-import Header from './Header';
+import Header from './Header.jsx';
```

Comment:
update webpack to handle jsx as default

Reply:
webpack is latest. It my bad, I'm used to writing the file extension(
can you tell me where I can get more information about this webpack feature?